packages/x-charts-pro/src/Heatmap/Heatmap.tsx

```diff
@@ -419,6 +419,10 @@ Heatmap.propTypes = {
           minSpan: PropTypes.number,
           minStart: PropTypes.number,
           panning: PropTypes.bool,
+          slider: PropTypes.shape({
+            enabled: PropTypes.bool,
+            size: PropTypes.number,
+          }),
```

Comment:
I don't think it can be used on heatmap can it?

Reply:
Yeah, it can't, but it's a bug in our types. The heatmap doesn't support zoom at all. 

I can take a look in a follow-up.

Reply:
We can probably just omit `slider` from `ChartContainerProProps` in `HeatmapProps`

Reply:
Here it is: https://github.com/mui/mui-x/pull/17657